bgpd/bgpd.h

```diff
@@ -965,7 +1066,8 @@ struct peer {
 #define PEER_FLAG_DEFAULT_ORIGINATE         (1 << 9) /* default-originate */
 #define PEER_FLAG_REMOVE_PRIVATE_AS         (1 << 10) /* remove-private-as */
 #define PEER_FLAG_ALLOWAS_IN                (1 << 11) /* set allowas-in */
-#define PEER_FLAG_ORF_PREFIX_SM             (1 << 12) /* orf capability send-mode */
```

Comment:
the comment moved out

Reply:
Same here i was getting more than 80 character warning from checkpactch here.. I know, It is strange